src/services/resource.js

```diff
@@ -210,6 +327,394 @@ module.exports = class resourceHelper {
 		}
 	}
 
+	static async upForReview(queryParams, tokenDetails, searchText = '', page, limit) {
+		try {
+			// get user details from token
+			const user_id = tokenDetails.id
+			const organization_id = tokenDetails.organization_id
+			const roles = tokenDetails.roles
+
+			let result = {
+				data: [],
+				count: 0,
+				in_progress_count: 0,
+			}
+
+			const sort = {
+				sort_by: queryParams.sort_by || common.CREATED_AT,
+				order: queryParams.sort_order?.toUpperCase() === common.SORT_ASC ? common.SORT_ASC : common.SORT_DESC,
+			}
+			let finalResourceIds = []
+			let resourceIdsToBeRemoved = []
+			let inProgressResources = []
+			let uniqueOrganizationIds = []
+
+			// check review resources and find all resources and org for the reviewer name.
+			const fetchReviewResourceDetails = await reviewResourcesQueries.findAll(
+				{
+					reviewer_id: user_id,
+				},
+				['organization_id']
+			)
+			uniqueOrganizationIds = utils.getUniqueElements(
+				fetchReviewResourceDetails.map((item) => item.organization_id)
```

Comment:
@adithyadinesh0412  if review resource is not there then you don't do this review fetch